MoviesMVVM/MoviesMVVM/NetworkService/NetworkService.swift

```diff
@@ -29,23 +30,26 @@ final class NetworkService: NetworkServiceProtocol {
 
     // MARK: - Private property
 
-    private let queryItemKey = URLQueryItem(
+    private lazy var queryItemKey = URLQueryItem(
         name: Constants.queryItemKeyName,
-        value: UserDefaults.standard.string(forKey: Constants.apiValueKeyName)
+        value: keychainService.getAPIKey(Constants.apiValueKeyName)
     )
-
     private let queryItemLanguage = URLQueryItem(
         name: Constants.queryItemLanguageName,
         value: Constants.queryItemLanguageValue
     )
 
     private var category = PurchaseEndPoint.popular
-
+    var keychainService: KeychainServiceProtocol
```

Comment:
```suggestion
    private var keychainService: KeychainServiceProtocol
```
Если не ругается, то private var